### ISSUE TITLE
System.OverflowException

### ISSUE THREAD

**Issue by Schaussi** [PandaWood/Simple-MAPI.NET#4]
Hi,

when I am testing SimpleMAPI in a fresh VS.NET project, everything works just fine.
Now I wanted to integrate it into another, larger project, but I get an exception there, even if there should not be any noticable difference to the test project.

Exception details (German):

> System.OverflowException
>    bei Win32Mapi.SimpleMapi.AllocRecips(Int32& recipCount)
>    bei Win32Mapi.SimpleMapi.Send(String subject, String noteText, Boolean ShowDialog)
>    bei xx.mailTo(String yyyy) in z:\zzzz\zzz.cs:Zeile 357.
>    ...
> Die arithmetische Operation hat einen Überlauf verursacht.

I use basically the same test code like in your Readme:

```
var mapi = new SimpleMapi();
mapi.AddRecipient(name: "bob@gmail.com", addr: null, cc: false);
//mapi.Attach(filepath: "c:\bob.txt")
mapi.Send(subject: "a subject", noteText: "a body text");
```

Could you give me a hint what could be the cause?

Thank you
  

(Edit: I am using the current NuGet version of your project)

**Comment by WacPat** [comment]
You are probably encountering the error when running it on a 64-bit machine or in a project that is compiled for 64-bit. I have just discovered this issue too a couple of days ago. Some of the memory allocations in SimpleMapi.net assume a 32-bit address space and when the allocation returns a 64-bit pointer and the code tries to cast it to a 32-bit value, it generates an Overflow exception.

I will attempt to fix the problem and push the patches to the maintainer.

**Comment by PandaWood** [comment]
I might need to fix the pull request for this (just merged) to reference this (eg "#4") to make it trackable
